src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -821,16 +829,16 @@ def actionCopy(self):
         Send a signal to the fitting perspective so parameters
         can be saved to the clipboard
         """
-        self.communicate.copyFitParamsSignal.emit("")
-        #self._workspace.actionPaste.setEnabled(True)
-        pass
+        if self._current_perspective is not None:
```

Comment:
good idea to replace signaling with direct call. Please update the method comment, otherwise it is confusing.
Same with the other methods below.

Reply:
Also, please remove the signal definitions from GUI Utilities

Reply:
Will do.